packages/edit-site/src/components/page-library/grid-item.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { BlockPreview } from '@wordpress/block-editor';
+import {
+	DropdownMenu,
+	MenuGroup,
+	MenuItem,
+	VisuallyHidden,
+	__experimentalHeading as Heading,
+	__experimentalHStack as HStack,
+	__unstableCompositeItem as CompositeItem,
+} from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { Icon, moreHorizontal } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import { store as reusableBlocksStore } from '@wordpress/reusable-blocks';
+
+/**
+ * Internal dependencies
+ */
+import { useLink } from '../routes/link';
+import { PATTERNS, USER_PATTERNS } from './use-patterns';
+
+const DeleteMenuItem = ( { item, onClose } ) => {
```

Comment:
I know things are pretty inconsistent in Gutenberg, and probably just personal preference, but top-level methods like this seem better suited to function definitions than a lambda.

Reply:
I'm happy to change it, I don't have a strong preference. For posterity though, rather than just "seem better suited", can you expand on why that is?

Reply:
nothing major:

- Functions are hoisted so ordering in the file is not important, you can refactor and move things around within the module without having to worry about the declaration order - which linting and testing is going to identify
- I  find function declarations quicker and easier to scan and differentiate from other constant declarations when at the module top level - definitely just a personal preference.
- Not an issue in this instance as the react dev tools assign the component name in the components browser, but in some instances using anon functions can make debugging tricker as stack traces may just show anon instead of a named method

I think @talldan had some thoughts on this (can't remember from which perspective though) - but if not don't feel that you need to change this just for my benefit.

Reply:
Thanks. 

I've switched out the arrow functions for the page library-related components. The util functions do seem to be more inconsistent in approach. I've kept those the same for now as it helps contrast just what changed in those vs the search functions in the block editor etc.